app/components/feedback-button/component.js

```diff
@@ -0,0 +1,68 @@
+import Component from '@ember/component';
+import microfeedback from 'microfeedback-button';
+import config from '../../config/environment';
+
+export default Component.extend({
+    didInsertElement() {
+        const buttonColor = this.get('buttonColor') || '#52A452';
+        let url = config.MICROFEEDBACK_URL;
+        // Add componentID to query params if provided
+        if (url) {
+            const params = {};
+            if (this.get('componentID')) {
+                params.componentID = this.get('componentID');
+            }
+            if (this.get('priorityID')) {
+                params.priorityID = this.get('priorityID');
+            }
+            const query = $.param(params);
+            url += `?${query}`;
+        }
+        const btn = microfeedback({
+            url,
+            backgroundColor: buttonColor,
+            showDialog: (btn) => {
+                const text = this.get('text') || '';
+                const placeholder = this.get('placeholder') || 'Describe your issue or share your ideas';
```

Comment:
I can make it so that the placeholder attribute isn't added if the `placeholder` component prop is falsy. I didn't initially add this because it complicated the code and didn't think it'd be a use case for us (all the placeholders for these feedback dialogs should be consistent anyway).

I can certainly implement that, though.